apps/prairielearn/src/tests/fileEditor.test.ts

```diff
@@ -806,36 +764,29 @@ function doEdits(data) {
     // (C, C, C*, C*)
 
     editPost('save_and_sync', data.contentsB, data.url, true, false, null);
-    // (C, C, C*, C*)
-
-    waitForJobSequence(locals, 'Error');
-    editPost('pull_and_save_and_sync', data.contentsB, data.url, true, false, null);
-    // (B, B, C, B)
-
     waitForJobSequence(locals, 'Success');
-    writeAndCommitFileInLive(data.path, data.contentsA);
-    // (B, A, C, B)
+    // (B, B, C*, B)
 
     editPost('save_and_sync', data.contentsA, data.url, true, false, null);
-    // (A, A, C, B)
+    waitForJobSequence(locals, 'Success');
+    // (A, A, C*, B)
```

Comment:
I think we should actually end up with the following here:

```suggestion
    // (A, A, C*, A)
```

Does that seem right? AFAICT, a successful `save_and_sync` should always end up with the first two elements and the last element of this tuple being equal (the same state).